### ISSUE TITLE
Crew Monitor Doesn't Update Fast

### ISSUE THREAD

**Issue by Rom546346** [HippieStationArchive/HippieStation13-1#3050]
I can look at it for a few seconds, memorize the location of the dead body, go into maint to get that dead body, find out there is no body, then go back to the monitor to see that none of the coordinates have changed. 

All this in two minutes. Seriously, that thing is practically useless.

**Comment by Spacedong** [comment]
get good and make sure no one took the body

**Comment by Swagile** [comment]
Keep it in your pocket and it constantly updates.

**Comment by Rom546346** [comment]
I haven't tried the pocket one, but I know for sure the console one is laggy as fuck.

**Comment by blobbernaut** [comment]
Yeah it's super fucking laggy

**Comment by Pyko1** [comment]
Yeah there is something iffy in it sometimes it pretty much updates every few seconds, and sometimes it just breaks up or updates really slow.

Trust me, I usually have one of them maximized on my second screen.
